### PR TITLE
Update how-to-deploy-the-app-v-51-server-using-a-script.md

### PR DESCRIPTION
**Pull request by baardhermansen**
Updated markdown for Notes.
Removed characters so that the tables are rendered correctly.

### CODE REVIEW

**Comment by illfated** [comment]
First of all, before I get into the messy part: Your changes look just fine, nothing appears to be wrong about them.
This PR is sort of interesting update, because now we are able to see how the HTML tables look in the rich diff preview.
So far, so good.

Have you looked at the HTML table contents yet? I have. They look pretty messy, at least when compared to each other.
I know, the HTML table contents are outside the scope of this PR, as far as I can tell.
Then again, I wonder: Should we leave that mess alone, or should we look into how much work remains to restore some order?
Never mind, I need to look at how the public page looks at the moment. I find it hard to believe it has been left alone looking like this.
I will get back to you as soon as I have looked up the public page and made up my mind about it.

**Comment by illfated** [comment]
Oh well. Seems the tables have looked the same as they do now, since 7 Apr 2016 and maybe even earlier. I'm too tired to even think of redesigning them or at least try to get them to make sense. 😞

- commit from 4 years ago showing the same HTML table content: https://github.com/MicrosoftDocs/windows-itpro-docs/commit/11674e7a1b647c59a373a515af6e6292b007d56e (direct file link with Rich diff view: https://github.com/MicrosoftDocs/windows-itpro-docs/commit/11674e7a1b647c59a373a515af6e6292b007d56e?short_path=47be641#diff-47be6413a4670ebae6a62463d867b4e1)
- commit adding the useless code block fence around the tables: https://github.com/MicrosoftDocs/windows-itpro-docs/commit/858476cf2f901f58d733f2943084bf01e11772c8 (direct file link with Rich diff view: https://github.com/MicrosoftDocs/windows-itpro-docs/commit/858476cf2f901f58d733f2943084bf01e11772c8?short_path=47be641#diff-47be6413a4670ebae6a62463d867b4e1)

Long story short: Looking at the contents in of the jumbled rectangles (1 sentence in the first column, dozens of lines in the second column, sometimes a table consists of only 1 row), it looks to me as if the content does not belong in this kind of table at all. I think the text would be more readable in ordinary bullet point lists with headings and sub-headings. 😔

I don't know if you have looked at the contents yet, but when you do, I would like to hear what you think of this oddly jumble-shaped content. 🤔

**Comment by baardhermansen** [comment]
I only took a quick peek at the tables prior to submitting the PR, but took a better look now. And i have to agree with you, the information does not render itself particularly well to a table format. 
I wonder if the new column format would be fitting for this? 
I'll have a stab at it and see if i can manage to make sense of it. If so i'll add a commit to this PR (unless it's been closed before i'm done).

**Comment by baardhermansen** [comment]
I'll open a separate PR, due to the major changes between this PR and the new one.

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by illfated** [comment]
@baardhermansen : Are you able to resolve the merge conflict in this PR file?

**Comment by baardhermansen** [comment]
My apologies, i should have closed this issue after 6280 was merged, i simply forgot :(
The differences between this PR and 6280 are too big to be resolved. 
Sorry again!

**Comment by illfated** [comment]
Thanks for the update. Good to have a closing comment with an explanation.
